.github/workflows/check-license.yml

```diff
@@ -3,6 +3,7 @@ name: License Verification
 on:
   pull_request_target:
     branches: [ main ]
+    paths: [ 'package.json' ]
```

Comment:
For reviewer: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths